articles/connections/database/custom-db.md

```diff
@@ -143,22 +159,21 @@ For example:
 callback(new ValidationError('email-too-long', 'Email is too long.'));
 ```
 
-## 6. Debug and troubleshoot
+## Troubleshoot
 
 Test the script using the **TRY** button. If your settings are correct you should see the resulting profile:
 
 ![](/media/articles/connections/database/mysql/db-connection-try-ok.png)
 
 If you add a `console.log` statement to the script you will be able to see the output here.
```

Comment:
Maybe enhance this a bit? Something like `If you do not get the expected result, you can use console.log statements in your script and try the connection again. The output of these statements will be printed in this window`